Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+JavaBindingsPrinting.swift

```diff
@@ -275,7 +283,7 @@ extension JNISwift2JavaGenerator {
   /// `func method(x: Int, y: Int) -> Int` becomes
   /// `long method(long x, long y)`
   private func renderFunctionSignature(_ decl: ImportedFunc) -> String {
-    let translatedDecl = translatedDecl(for: decl)
+    let translatedDecl = translatedDecl(for: decl)!
```

Comment:
Are we sure about the !, can we either make a guard with a better message or document why the ! Should not happen